contrib/api-apache.conf

```diff
@@ -1,16 +1,15 @@
-# SPDX-FileCopyrightText: 2020 Birger Schacht
-#
+# SPDX-FileCopyrightText: 2022 CERT.at GmbH <https://cert.at/>
 # SPDX-License-Identifier: CC0-1.0
- 
-#Override the default configuration file path using the
-#INTELMQ_API_CONFIG environment variable
-#SetEnv INTELMQ_API_CONFIG /etc/intelmq/api-config.json
-<IfModule mod_wsgi.c>
-WSGIApplicationGroup %{GLOBAL}
-WSGIPassAuthorization On
-WSGIScriptAlias /intelmq /usr/lib/python3/dist-packages/intelmq_api/intelmq-api.wsgi
-</IfModule>
 
-<Directory /usr/lib/python3/dist-packages/intelmq_api/>
-   Require all granted
-</Directory>
+
+<VirtualHost *:80>
```

Comment:
This file is/was not a vhost definition, in other words, it's not placed in `/etc/apache2/sites-available/`, but in `/etc/apache2/conf-available/` (Debian-style paths), independent of the vhost definition and thus independent of any HTTPS vhost definition.